src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -126,6 +128,10 @@ public void createDataset(Dataset dataset) {
         }
     }
 
+    public static final String GetLoadTableName(String datasetName){
+        return datasetName + "_load";
+    }
+
```

Comment:
Looking for suggestions here:
1) Does this name make sense? Any other name suggestions?
2) Does this make sense to live in a helper method like this? It's public so it can be shared with the test. I considered a few other options, such as adding it as a field attached to the DatasetSummary object, but that seemed like overkill.

Reply:
We reserve the "datarepo_" names for ourselves. Something made with the datasetName might conflict with a user table.

Since there is exactly one of these in the dataset, it can be a fixed constant. You can build it in PdaoConstant.java. There are several other ones there. And then skip the method.

I'd suggest: PDAO_PREFIX + "load_history"